src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -2503,6 +2503,46 @@ class npc_bloodmage_laurith : public CreatureScript
     }
 };
 
+enum OrabusTheHelmsman
+{
+    NPC_ORABUS            = 32576,
+    NPC_KVALDIR_CREWMAN_1 = 32577,
+    NPC_KVALDIR_CREWMAN_2 = 32578,
+    NPC_KVALDIR_CREWMAN_3 = 32579,
+    NPC_KVALDIR_CREWMAN_4 = 32580
+};
+
+class npc_orabus_the_helmsman_ship_exit_pos : public UnitScript
+{
+public:
+    npc_orabus_the_helmsman_ship_exit_pos() : UnitScript("npc_orabus_the_helmsman_ship_exit_pos") { }
+
+    void ModifyVehiclePassengerExitPos(Unit* passenger, Vehicle* /*vehicle*/, Position& pos)
+    {
+        if (passenger->GetTypeId() == TYPEID_UNIT)
+        {
+            switch (passenger->GetEntry())
+            {
+            case NPC_ORABUS:
+                pos.Relocate(2807.858f, 7038.571f, 7.075813f, 4.73481f);
```

Comment:
are these supposed to be hardcoded or is there any offset that might make more sense ?

Reply:
I thought hardcoded made more sense as the boat is always on the same position so the exit pos will always be the same regardless.

[I found a sniff](https://gist.github.com/Wyrserth/8f7e650d25f65eefb31f1db17b93b992) lying around and all the coordinates match with retail (I actually made a mistake and inverted the coordinates of some npc entries that were correct in the original script, fixing it now).

So I would leave them hardcoded as I don't think I can replicate the same coordinates with offsets, if that's alright.